spaceR/spaceR/ViewModels/RocketViewModel.swift

```diff
@@ -25,14 +25,14 @@ class RocketViewModel {
         self.allRockets = []
     }
     
-    var rocketList: [Rocket] {
-        return allRockets ?? []
-    }
-    
     var rocketCount: Int {
         return allRockets?.count ?? 0
     }
     
+    func getRocket(atIndex: Int) -> Rocket? {
```

Comment:
We try to avoid using the prefix "get" we ideally we want this function to read like a sentence i.e. 

`func rocket(at index: Int) -> Rocket? {`